client/src/main/java/se/fortnox/reactivewizard/client/SingleWithResponse.java

```diff
@@ -24,4 +24,8 @@ HttpClientResponse getResponse() {
         }
         return httpClientResponse.get();
     }
+
+    static <T> SingleWithResponse<T> from(SingleWithResponse<T> singleWithResponse, Single<T> inner) {
```

Comment:
these  have no defined scope, where are these going to be used?

Reply:
It should be used In our subsystem but should not easily used, thus the package private scope.